benchmarks/spark/spark_nyc_taxi_preciptation.py

```diff
@@ -0,0 +1,115 @@
+import time
+
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import (
+    col,
+    dayofweek,
+    hour,
+    month,
+    to_date,
+)
+
+
+def get_monthly_travels_weather():
+    spark = (
+        SparkSession.builder.appName("MonthlyTravelsWeather")
+        .config("spark.jars.packages", "org.apache.hadoop:hadoop-aws:3.3.2")
+        .config("spark.sql.execution.arrow.pyspark.enabled", "true")
+        .config(
+            "fs.s3a.aws.credentials.provider",
+            "org.apache.hadoop.fs.s3a.AnonymousAWSCredentialsProvider",
+        )
+        .getOrCreate()
+    )
+    import pyspark.pandas as ps
```

Comment:
Any problem with moving the import to the top?

Reply:
Nope, moved